src/main/kotlin/org/partiql/scribe/sql/RelConverter.kt

```diff
@@ -1,269 +1,468 @@
 package org.partiql.scribe.sql
 
-import org.partiql.ast.Expr
+import org.partiql.ast.Ast.exclude
```

Comment:
self-review: support all the previous PartiQL -> PartiQL transpilation features. We still do not yet have support for:
- CTEs
- LIMIT
- OFFSET
- SORT (ORDER BY)
- HAVING clause
- INTERVAL data type

Reply:
Perhaps we should have open issues for these?

Reply:
Yeah, I will cut issues.